### ISSUE TITLE
uboot: 2023.07.02 -> 2024.01

### ISSUE THREAD

**Issue by TravisWhitaker** [NixOS/nixpkgs#309137]
### Steps To Reproduce
1. ...
2. ...
3. ...

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
output here
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by TravisWhitaker** [comment]
This problem doesn't exist in u-boot 2023.10.

**Comment by flokli** [comment]
@TravisWhitaker can you bisect and take this to the U-Boot mailing list, and link here? Doesn't seem to be a nixpkgs-specific issue.

**Comment by TravisWhitaker** [comment]
Yes this will need to be fixed upstream in u-boot, but the fact that is NixOS-specific is that 23.11's `ubootQemuAarch64` works with XHCI and master's/presumably 24.05's will not (so finding this issue might wind up helping someone out).
